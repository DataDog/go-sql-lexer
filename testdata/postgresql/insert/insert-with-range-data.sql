INSERT INTO reservations (during) VALUES ('[2023-01-01 14:00, 2023-01-01 15:00)');
