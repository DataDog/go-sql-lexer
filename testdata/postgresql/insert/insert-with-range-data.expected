[
    {
        "expected": "INSERT INTO reservations ( during ) VALUES ( ? )",
        "statement_metadata": {
            "size": 18,
            "tables": [
                "reservations"
            ],
            "commands": [
                "INSERT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]